test/e2e/feature/feature.go

```diff
@@ -71,6 +71,7 @@ var (
 	KubeProxyDaemonSetDowngrade             = framework.WithFeature(framework.ValidFeatures.Add("KubeProxyDaemonSetDowngrade"))
 	KubeProxyDaemonSetUpgrade               = framework.WithFeature(framework.ValidFeatures.Add("KubeProxyDaemonSetUpgrade"))
 	KubeProxyDaemonSetMigration             = framework.WithFeature(framework.ValidFeatures.Add("KubeProxyDaemonSetMigration"))
+	KubeUp                                  = framework.WithFeature(framework.ValidFeatures.Add("KubeUp"))
```

Comment:
name SGTM. KubeUp is technically not a feature, it's a legacy deployer but the intention is clear.

Reply:
The intention was to add comments for new features. That we don't have those for legacy features is unfortunate. We shouldn't dig ourselves in deeper by not explaining new features either.